devito/types/relational.py

```diff
@@ -0,0 +1,255 @@
+"""User API to specify relationals."""
+
+import sympy
+
+__all__ = ['Le', 'Lt', 'Ge', 'Gt', 'Ne']
+
+
+class Le(sympy.Le):
+    """
+    A less-than or equal ("<=") relation between two objects, the left-hand side and the
+    right-hand side. Can be used to build conditionals but not directly to
+    construct an Operator.
+
+    The left-hand side may be a Function or a SparseFunction. The right-hand
```

Comment:
Drop this whole paragraph, useless (clearly apply to all other docstrigns too)

Reply:
Dropped, applied.